ui/cypress/integration/AccessDocumentation.feature

```diff
@@ -0,0 +1,4 @@
+Feature: AccessDocumentation
+ Scenario: Access documentation from the Navbar
+     Given I log in
+     Then I access the documentation from the navbar
```

Comment:
nitpick: `Then`s are more about verifying some fact, not performing an action. Wording could reflect this, by adjusting it slightly to:

```gherkin
    Then I can access the documentation from the Navbar
```

Reply:
It's definitely a good point to improve!!!